src/components/table/TablePagination.js

```diff
@@ -221,8 +222,10 @@ class TablePagination extends Component {
                     }}
                 >
                     {selectedWholePage ?
-                        'Select all ' + size + ' items' :
-                        'Select all on this page'
+                        counterpart.translate(
+                            'view.selectAll.caption'
+                        ) + size + ' items' :
```

Comment:
can we use some parameterized message mechanism.

For example in java we have frameworks which can accept a message like 
```
view.selectAll.caption=Select {0} items
```
and then you could use it something like:
```
counterpart.translate('view.selectAll.caption', size)
```

I am quite sure we have some alternatives in javascript too (might be with a different placeholder).

I would highly suggest to use something like that because else, messages like that cannot be translated. Imagine that different languages have different sentence structure.